app/src/test/java/com/example/p2pchat/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.example.p2pchat;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() {
+        assertEquals(4, 2 + 2);
```

Comment:
Looks like Android Studio has unit testing built right in. That just made one of my issues much easier.